.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Set the path of the Flutter SDK to the path of the FVM Flutter version.
+    //
+    // VS Code will always use the version selected within the project for all
+    // IDE tooling.
+    //
+    // Source: https://fvm.app/docs/getting_started/configuration#option-1---automatic-switching-recommended
+    "dart.flutterSdkPath": ".fvm/flutter_sdk",
```

Comment:
Android Studio users need to set up FVM for themselves in Android Studio, because I think in Android Studio isn't something like `settings.json` which can commit to git (I think the `.idea/workspace.xml` contains also information about the local machine): https://fvm.app/docs/getting_started/configuration/#android-studio

Reply:
Thanks for the info, then we should add this to our setup documentation doc as well